GetF!t/Modules/History/HistoryViewModel.swift

```diff
@@ -5,6 +5,7 @@ struct ExerciseDay: Identifiable {
     let id = UUID()
     let date: Date
     var exercises: [String] = []
+    var rating: [Int] = []
```

Comment:
nebolo by lepšie tu mať pole tuplov?
```suggestion
    var exercises: [(String, Int)] = []
```

Reply:
Potom by si mal vždy zaručené, že máš rovnako ratingov ako exercisov..